plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/ModulesManager.java

```diff
@@ -1171,6 +1177,29 @@ public AdditionalGrammarVersionsToCheck getAdditionalGrammarVersions()
 
                     }
                 }
+            } else if ("typing".equals(n.getName())) {
+                Module module = (Module) ((SourceModule) n).getAst();
+                for (SimpleNode node : module.body) {
+                    if (node instanceof Assign && ((Assign) node).value instanceof Call
+                            && ((Call) ((Assign) node).value).func instanceof Name
+                            && ((Name) ((Call) ((Assign) node).value).func).id.equals("_alias")) {
```

Comment:
Must check if `id == null` (so, compare the other way around -- with `"_alias".equals(...)` to it so that you don't have an NPE).